### DISCUSSION TITLE
LastBen Colab for A1111 Int images not working?

### DISCUSSION THREAD

**Discussion by OrganicBeej** [AlUlkesh/sd_save_intermediate_images#27] [Q&A]
Hi all,

This was working fine last night but today, doesnt seem to be working at all?
I've checked the vsn and it seems up to date?
<img width="899" alt="image" src="https://user-images.githubusercontent.com/77268953/225900677-3a246c8f-5c41-4232-be08-75e84b8aff1e.png">
Any ideas?

Thanks :)

**Comment by OrganicBeej** [comment]
Apologies, did a fresh re-start and all seems well again?? 🤷🏼‍♂️😊
I love this extension, it's wonderful to track these earlier renders and sometimes mix them together 👍🏻🌅

**Reply by AlUlkesh** [reply]
Thanks. Sometimes webui acts weird when you do an update from the ui...
